docarray/array/qdrant.py

```diff
@@ -5,8 +5,40 @@
 
 
 class DocumentArrayQdrant(StorageMixins, DocumentArray):
-    """This is a :class:`DocumentArray` that uses Qdrant as
-    vector search engine and storage.
+    """
+    DocumentArray that stores Documents in a `Qdrant <https://weaviate.io/>`_ vector search engine.
+
+    .. note::
+        This feature requires `qdrant-client`. You can install it via `pip install "docarray[qdrant]"`.
```

Comment:
I would not say it is a feature maybe class or object ?


```suggestion
        This DocumentArray requires `qdrant-client`. You can install it via `pip install "docarray[qdrant]"`.
```